credentials/src/test/resources/pd_filter_array.json

```diff
@@ -12,10 +12,7 @@
             ],
             "filter": {
               "type": "string",
-              "contains": {
-                "type": "string",
-                "const": "StreetCred"
-              }
+              "pattern": ".*StreetCred.*"
```

Comment:
+1

Reply:
It was not valid, contains should only be used for array
for type string its pattern
              
https://json-schema.org/understanding-json-schema/reference/array

This will be caught and prevented in the next PR with PD validation + test vectors

Reply:
Ah I think this was originally expecting the value at the path to be a string array, which is why `contains` + `type: string` was used. 

```
 "$.vc.type[*]",
 "$.type[*]"
```

If the value at these paths is a single string and not a string array, then yeah we need pattern.